services/ml/compilation_impl_mac.mm

```
@@ -250,8 +250,11 @@
     } else if (type == mojom::FULLY_CONNECTED) {
       success = CompileFullyConnected(mps_image_nodes_, operation, operands_,
                                       values_, memory_);
+    } else if (type == mojom::RESIZE_BILINEAR) {
+      success = CompileBilinearScale(mps_image_nodes_, operation, operands_,
+                                     values_, memory_);
     } else {
-      DLOG(ERROR) << "Operation is not supported";
+      NOTREACHED();
```

Does it mean GPU process will crash when the op is not supported? Should we report an error instead?

---

Done with `success = false`